test/functional/private_groups.js

```diff
@@ -910,6 +910,85 @@ describe("PrivateGroups", function() {
     })
   })
 
+  describe('subscribers', function(){
+    let adminContext = {}
+      , plutoContext = {}
+      , marsContext = {}
+      , group
+
+    beforeEach(funcTestHelper.createUserCtx(adminContext, 'Luna', 'password'))
+    beforeEach(funcTestHelper.createUserCtx(marsContext, 'Mars', 'wordpass'))
+    beforeEach(funcTestHelper.createUserCtx(plutoContext, 'Pluto', 'wordpass'))
+
+    beforeEach(async () => {
+      const response = await funcTestHelper.createGroupAsync(adminContext, 'pepyatka-dev', 'Pepyatka Developers')
+      group = response.group
+
+      await funcTestHelper.subscribeToAsync(plutoContext, group)
+
+      await funcTestHelper.groupToPrivate(group, adminContext)
+
+      await funcTestHelper.createAndReturnPostToFeed(group, adminContext, 'Post body')
+    })
+
+    it('anonymous users should have no access to private group subscribers', async function(done){
```

Comment:
you use the same `async function(done)` pattern in tests. it's not needed. async works without done. so, all you need is actually `async () => `